### ISSUE TITLE
autodispose-rxlifecycle3 is not published

### ISSUE THREAD

**Issue by MarkyC** [uber/AutoDispose#338]
In the readme, there is a mvnrepository link for rxlifecycle3, but this package does not exist, leaving us with no way to use `autodispose-rxlifecycle3` (short of cloning [this repo](https://github.com/uber/AutoDispose/tree/master/autodispose-rxlifecycle3)):

`autodispose-rxlifecycle3` [![Maven Central](https://img.shields.io/maven-central/v/com.uber.autodispose/autodispose-rxlifecycle3.svg)](https://mvnrepository.com/artifact/com.uber.autodispose/autodispose-rxlifecycle3)
```gradle
implementation 'com.uber.autodispose:autodispose-rxlifecycle3:x.y.z'
```

**Library version**:

**Repro steps or stacktrace**:

**Comment by ZacSweers** [comment]
Yes sorry it hasn't been released yet. Will try to take care of it soon!

**Comment by ZacSweers** [comment]
Released in 1.2.0!
